lib/src/components/NxDropdownMenu/NxDropdownMenu.scss

```diff
@@ -102,6 +106,23 @@
       cursor: default;
     }
   }
+
+  &:focus {
```

Comment:
The other focus block in this file now effectively only applies to `.nx-dropdown-button` and not also `.nx-dropdown-link`.  Let's move it out to be specific to `.nx-dropdown-button`.

Reply:
Updated here 694650c86648fc123ff136c037e180f703fd59d3.